homeassistant/components/stream/core.py

```diff
@@ -155,14 +158,17 @@ class StreamOutput:
     """Represents a stream output."""
 
     def __init__(
-        self, hass: HomeAssistant, idle_timer: IdleTimer, deque_maxlen: int = None
+        self,
+        hass: HomeAssistant,
+        idle_timer: IdleTimer,
+        deque_maxlen: int | None = None,
     ) -> None:
         """Initialize a stream output."""
-        self._hass = hass
-        self._idle_timer = idle_timer
+        self._hass: HomeAssistant = hass
+        self._idle_timer: IdleTimer = idle_timer
         self._sequence_cursor: int = -1
-        self._event = asyncio.Event()
-        self._part_event = asyncio.Event()
+        self._event: asyncio.Event = asyncio.Event()
+        self._part_event: asyncio.Event = asyncio.Event()
         self._segments: deque[Segment] = deque(maxlen=deque_maxlen)
```

Comment:
But not for `deque`.
```suggestion
        self._event = asyncio.Event()
        self._part_event = asyncio.Event()
        self._segments: deque[Segment] = deque(maxlen=deque_maxlen)
```